src/test/java/io/zold/api/WalletTest.java

```diff
@@ -61,10 +56,15 @@ public void readsWalletId() throws IOException {
         MatcherAssert.assertThat(wallet.id(), Matchers.is(id));
     }
 
-    @Ignore
     @Test
-    public void throwIoExceptionIfReadingIdFails() throws IOException {
-        this.error.expect(IOException.class);
+    public void throwRuntimeExceptionIfReadingIdFails() throws IOException {
+        this.error.expect(RuntimeException.class);
```

Comment:
@paulodamaso let's expect a narrower exception than `RuntimeException`. Furthermore, let's on check the exception message with `this.error.expectMessage` as well.

Reply:
@carlosmiranda All right